Makefile

```diff
@@ -13,3 +13,8 @@ security_check:
 lint:
 	pipenv run flake8
 .PHONY: lint
+
+init:
+	pip3 install pipenv
```

Comment:
This is a bit problematic. First, you can install `pipenv` using other means (e.g. `brew`) and you might already have it.

Secondly, `pip3` implies you have Python 3 installed as `python3` and `pip3`. Which may not be the case. I would explain that the project depends on `pipenv` in the README (well, we'll do it anyway) so people can install it the way they think it's best for them.

Reply:
this ceremony is that required by Travis CI for their base image. If you think this is confusing to other users we could rename it `travis-ci-init` or something?

Reply:
I would rather have it directly in `.travis.yml` then.